web/webapi.py

```diff
@@ -420,7 +421,9 @@ def dictify(fs):
                 # since wsgi.input is directly passed to multipart,
                 # it can not be called multiple times. Saving the result
                 # object in ctx to allow calling web.input multiple times.
-                a = ctx.get("_fieldstorage")  # TODO: Rename? is this visible anywhere else?
+                a = ctx.get(
+                    "_fieldstorage"
+                )  # TODO: Rename? is this visible anywhere else?
```

Comment:
Please do not wrap lines just to make room for a comment.
```suggestion
                # TODO: Rename?  Is this visible anywhere else?
                a = ctx.get("_fieldstorage")
```

Reply:
Those changes were made by `ruff --fix`, not me.